### PR TITLE
Update most dependencies to fix security vulnerabilities

### PR DESCRIPTION
**Pull request by tomap**

### CODE REVIEW

**Review by curbengh — DISMISSED** [review]

**Comment by curbengh** [comment]
Conflicts with https://github.com/hexojs/hexo-webui/pull/23.

**Comment by curbengh** [comment]
thanks for rebasing. suggest to add `^` to all versions and include https://github.com/hexojs/hexo-webui/pull/23 in this PR.

**Comment by tomap** [comment]
@curbengh I've fixed the versions with `^` and included #23 
Rebase will be done by github during squash merged :)

File: package.json [hunk 1]
```diff
@@ -5,12 +5,12 @@
   "main": "lib/index",
   "repository": {
     "type": "git",
-    "url": "git://github.com/tommy351/hexo-webui.git"
+    "url": "git://github.com/hexo/hexo-webui.git"
```
Comment (curbengh) [inline package.json]: "hexojs/hexo-webui"

File: package.json [hunk 2]
```diff
@@ -5,12 +5,12 @@
...
   },
   "bugs": {
-    "url": "https://github.com/tommy351/hexo-webui/issues"
+    "url": "https://github.com/hexo/hexo-webui/issues"
```
Comment (curbengh) [inline package.json]: hexojs

**Comment by tomap** [comment]
@curbengh & @YoshinoriN ready to be merged :)

**Review by curbengh — APPROVED** [review]

**Pull request merged**